esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2335,6 +2384,7 @@ bool CFileSprayEx::onDespray(IEspContext &context, IEspDespray &req, IEspDespray
         PROGLOG("Despray %s", srcname);
```

Comment:
Is there a jira for gordon to update eclwatch to provide the name?

Reply:
No changes needed in eclwatch.  This field is already being provided by eclwatch.  It just wasn't defined in the ecm.